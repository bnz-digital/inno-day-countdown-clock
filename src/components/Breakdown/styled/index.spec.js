import React from 'react'
import { shallow } from 'enzyme'

import StyledBreakdown from '.'

describe('components:Breakdown:styled', () => {
  it('renders <StyledBreakdown/> correctly', () => {
    expect(toJson(shallow(<StyledBreakdown />))).toMatchSnapshot()
  })
})
