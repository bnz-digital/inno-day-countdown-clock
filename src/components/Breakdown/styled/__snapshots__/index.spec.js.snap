// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:Breakdown:styled renders <StyledBreakdown/> correctly 1`] = `
.c0 {
  color: #FAA61B;
  font-size: 4vw;
  line-height: 1.5;
  position: fixed;
  bottom: 0;
  left: 0;
  right: 0;
  padding: 1vw;
  text-align: center;
}

<footer
  className="c0"
/>
`;
